README.md

```diff
@@ -182,3 +182,30 @@ await knockClient.preferences.setWorkflow("dinosaurs-loose", {
   },
 });
 ```
+
+
+### Managing the current user's channel data
+
+```typescript
+// Get user channel data
+await knockClient.user.getChannelData({
+  userId: "user-id",
+  channelId: "channel-id"
+});
+```
+
+```typescript
+// Set push channel data for a user
+await knockClient.user.setChannelData({
+  userId: "user-id",
```

Comment:
In code I don't see that you pass `userId` to this method, I am not sure if this is bug in just README file or in code?